src/python/pants/option/global_options.py

```diff
@@ -433,6 +433,14 @@ def register_bootstrap_options(cls, register):
             "Patterns use the gitignore syntax (https://git-scm.com/docs/gitignore). "
             "The `--pants-distdir` and `--pants-workdir` locations are inherently ignored.",
         )
+        register(
+            "--pants-gitignore",
+            advanced=True,
+            type=bool,
+            default=False,
```

Comment:
What's the rationale for defaulting to False?

Reply:
+1 .. should default to on, assuming it works in the absence of a `.gitignore` file.